pkg/rpc/client/rpc.go

```diff
@@ -597,3 +613,28 @@ func (c *Client) AddNetworkFee(tx *transaction.Transaction, extraFee int64, accs
 func (c *Client) GetNetwork() netmode.Magic {
 	return c.network
 }
+
+// GetNativeContractHash returns native contract hash by its name. It is not case-sensitive.
+func (c *Client) GetNativeContractHash(name string) (util.Uint160, error) {
+	lowercasedName := strings.ToLower(name)
+	switch lowercasedName {
+	case "neo", "gas":
+		if c.initDone {
+			return c.cache.nativeHashes[lowercasedName], nil
+		}
+
+	case "policy", "oracle", "designation":
+		hash, ok := c.cache.nativeHashes[lowercasedName]
+		if ok {
+			return hash, nil
+		}
+	default:
+		return util.Uint160{}, fmt.Errorf("unknown native contract name: %s", name)
+	}
+	cs, err := c.GetContractStateByAddressOrName(lowercasedName)
```

Comment:
And it's better to use `name` here, just in case (not changing the requested name).